changelog_unreleased/blog-post-intro.md

```diff
@@ -0,0 +1,13 @@
+---
+author: "Sosuke Suzuki (@sosukesuzuki)"
+authorURL: "https://github.com/sosukesuzuki"
+title: "Prettier 2.6: new singleAttributePerLine option and new JavaScript features!"
+---
+
+This release includes a new `singleAttributePerLine` option. For us, adding a new option was a hard decision.
```

Comment:
Can you add a few sentences here to describe why the new option is necessary?

Reply:
Okay, can you review the below sentence?

***

This release includes a new `singleAttributePerLine` option. This is an option to print single attribute per line in HTML, JSX, Vue SFC template. Prettier, following its [Option Philosophy](https://prettier.io/docs/en/option-philosophy.html), does not like to add such an option. However, there are many users who want this feature, and indeed [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript) and [eslint-plugin-vue](https://eslint.vuejs.org/) recommend the singe attribute per line style. A PR ([#6644](https://github.com/prettier/prettier/pull/6644)) to add this feature was opened in October 2019 and has been supported by many until now. For us it was a hard decision to add this option. We hope the addition of this option will benefit many users.

Reply:
I made a few grammar tweaks (and linked the style guide references to the definition of the rule) but otherwise it looks great!

> This release includes a new `singleAttributePerLine` option. This is an option to print only one attribute per line in Vue SFC templates, HTML, and JSX. Per our [Option Philosophy](https://prettier.io/docs/en/option-philosophy.html), we would prefer not to add such an option. However, there are many users who want this feature, and major style guides like [Airbnb’s JavaScript Style Guide](https://github.com/airbnb/javascript/blob/274c8d570155a05b016980294d4204c5711bce86/packages/eslint-config-airbnb/rules/react.js#L97-L99) and [Vue’s style guide](https://vuejs.org/style-guide/rules-strongly-recommended.html#multi-attribute-elements) recommend the single attribute per line style. A PR (https://github.com/prettier/prettier/pull/6644) to add this feature was opened in October 2019, and both it and the corresponding issue (https://github.com/prettier/prettier/issues/5501) have received a significant amount of support from users. For us it was a hard decision to add this option. We hope the addition of this option will benefit many users without significantly harming our principles.


```suggestion
This release includes a new `singleAttributePerLine` option. This is an option to print only one attribute per line in Vue SFC templates, HTML, and JSX. Per our [Option Philosophy](https://prettier.io/docs/en/option-philosophy.html), we would prefer not to add such an option. However, there are many users who want this feature, and major style guides like [Airbnb’s JavaScript Style Guide](https://github.com/airbnb/javascript/blob/274c8d570155a05b016980294d4204c5711bce86/packages/eslint-config-airbnb/rules/react.js#L97-L99) and [Vue’s style guide](https://vuejs.org/style-guide/rules-strongly-recommended.html#multi-attribute-elements) recommend the single attribute per line style. A PR (https://github.com/prettier/prettier/pull/6644) to add this feature was opened in October 2019, and both it and the corresponding issue (https://github.com/prettier/prettier/issues/5501) have received a significant amount of support from users. For us it was a hard decision to add this option. We hope the addition of this option will benefit many users without significantly harming our principles.
```

Reply:
thank you!